StudentsDB/src/test/java/com/musala/simple/students/db/MainTest.java

```diff
@@ -0,0 +1,145 @@
+package com.musala.simple.students.db;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.musala.simple.students.db.exceptions.SQLConnectionException;
+import com.musala.simple.students.db.exceptions.StudentDataParsingException;
+
+public class MainTest {
+
+	final static Logger LOGGER = LoggerFactory.getLogger(MainTest.class);
+
+	private ClassLoader classLoader = new Main().getClass().getClassLoader();
+	private File file = new File(classLoader.getResource("myData.json").getFile());
+	private final String testPath = file.getAbsolutePath();
+
+	@Test
+	public void testPassingOneArgument() {
+		String[] args = { testPath };
+		try {
+			Main.main(args);
+			assertTrue(true);
+		} catch (StudentDataParsingException | SQLConnectionException e) {
+			e.printStackTrace();
```

Comment:
No need to print stack trace in tests - maven will do this for you ;) Please remove stack trace printing throughout tests.

Reply:
Removed.